src/reporting/reporters/internal/scanner.ts

```diff
@@ -31,9 +31,7 @@ export function reportScannerLoggerEvents(logger: Logger): void {
    */
   let spinner: Spinner;
   if (process.stdout.isTTY) {
-    spinner = new Spinner({
-      text: startMessage.message
-    });
+    spinner = new Spinner().start(startMessage.message);
```

Comment:
```suggestion
    spinner = new Spinner();
```
 `spinner.start();` is at line 44 below, where you need to update into `.start(startMessage.message)`

Reply:
oh my bad ><
Updated